app/controllers/customer/delete.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    deleteCustomer: function (customer_id) {
```

Comment:
so in new, this doens't take an argument, and then does

      let customer = this.get('model').save();

I wonder if that approach to fetching the model works? and then you don't pass anything when you call `deleteCustomer`